hack/make/ubuntu

```diff
@@ -91,6 +95,10 @@ if [ "$1" = 'configure' ] && [ -z "$2" ]; then
 	fi
 fi
 
+if [ -x /sbin/apparmor_parser ]; then
+	/sbin/apparmor_parser -r -W -T /etc/apparmor.d/ docker
```

Comment:
Also, shouldn't this only be run `if aa-status --enabled 2>/dev/null` ?

(ref https://sources.debian.net/src/apparmor/2.9.2-3/debian/debhelper/postinst-apparmor/#L18)